scripts/utils/pre-release.mjs

```diff
@@ -0,0 +1,370 @@
+  }
+}
+
+/**
+ * Verify if the builds of all of the packages defined as workspaces have correct version number in them.
+ * Currently it's checking the following builds:
+ * - the one declared as default,
+ * - UMD (if it's declared under the 'jsdelivr' key in the package.json file or a `umd` key in this function's
+ * settings).
+ */
+export async function verifyBundles() {
+  const packagesInfo = {
+    handsontable: {
+      className: 'Handsontable',
+      umd: 'tmp/dist/handsontable.full.min.js',
+      entryFile: 'tmp/index.mjs',
+      defaultExport: true
+    },
+    '@handsontable/angular': {
+      className: 'HotTableModule'
+    },
+    '@handsontable/react': {
+      className: 'HotTable'
+    },
+    '@handsontable/vue': {
+      className: 'HotTable'
+    }
+  };
+  const hotPackageJson = require('../../package.json');
+  const mismatchedVersions = [];
+  JSDOMGlobal();
+
+  console.log(`\nMain package.json version:\n${chalk.green(hotPackageJson.version)}\n`);
+
+  for (const packagesLocation of workspacePackages) {
+    const subdirs = glob.sync(packagesLocation);
+
+    for (const subdir of subdirs) {
+      const packageJsonLocation = `../../${subdir}/package.json`;
+      const packageJson = require(packageJsonLocation);
+      const packageName = packageJson.name;
+      const defaultPackage = await import(
+        packagesInfo[packageName].entryFile ?
+          `../../${subdir}/${packagesInfo[packageName].entryFile}` :
+          packageName
+        );
+      let defaultPackageVersion = null;
+      let umdPackageVersion = null;
+      let umdPackage = null;
+
+      if (packagesInfo[packageName].umd || packageJson.jsdelivr) {
+        umdPackage = await import(
+          packagesInfo[packageName].entryFile ?
+            `../../${subdir}/${packagesInfo[packageName].umd}` :
+            `${packageName}/${packageJson.jsdelivr}`
+          );
+        umdPackage = umdPackage.default;
+      }
+
+      if (packagesInfo[packageName]?.defaultExport) {
+        defaultPackageVersion = defaultPackage.default.version;
+
+      } else {
+        defaultPackageVersion = defaultPackage[packagesInfo[packageName]?.className]?.version;
+
+        if (umdPackage) {
+          umdPackageVersion = umdPackage[packagesInfo[packageName]?.className]?.version;
+        }
+      }
+
+      if (hotPackageJson.version !== defaultPackageVersion) {
+        mismatchedVersions.push(`${packageName} (default) - ${defaultPackageVersion}`);
+      }
+
+      if (umdPackageVersion && (hotPackageJson.version !== umdPackageVersion)) {
+        mismatchedVersions.push(`${packageName} (UMD) - ${umdPackageVersion}`);
+      }
+    }
+  }
+
+  if (mismatchedVersions.length > 0) {
+    mismatchedVersions.forEach((mismatch) => {
+      displayErrorMessage(`\nMismatched versions in ${mismatch}.`);
+    });
+
+    process.exit(1);
+
+  } else {
+    displayConfirmationMessage('\nAll packages have the expected version number.\n');
+
+    process.exit(0);
```

Comment:
This line prevents the `freeze` script from finishing its commands. The line should be removed.
```suggestion
```

Reply:
It was a workaround for this issue: https://github.com/facebook/react/issues/20756, but it only works if the script is run as a separate process, so I moved it back to it's own separate file.

Reply:
Thanks for explanation 👍